packages/cadl-vs/VS2019/Microsoft.Cadl.VS2019.csproj

```diff
@@ -14,7 +14,7 @@
     <!-- Use 16.latest for build tools -->
     <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="16.11.65" PrivateAssets="All" />
     <!-- Align with VS 16.0 version here: https://devblogs.microsoft.com/visualstudio/using-newtonsoft-json-in-a-visual-studio-extension/-->
-    <PackageReference Include="NewtonSoft.JSON" Version="9.0.1" ExcludeAssets="Runtime" />
+    <PackageReference Include="NewtonSoft.JSON" Version="13.0.2" ExcludeAssets="Runtime" />
```

Comment:
I guess we probabl.y don't need to support VS 16.*0* even if we keep VS 2019 support so maybe we could use 12.0.latest here and still get 16.2+ compatibility according to that table, but I think the best would be to remove the VS2019 support altogether, but we should decide and do that separately.

Reply:
As far as compilance, though, this is a false positive. We don't ship NewtonSoft.JSON, we just reference it and the code that executes is determined by what VS ships in the box. See `ExcludeAssets="Runtime"`. So what matters is that the user has a supported, compliant VS version, and we do not influence that. Is there a suppression mechanism?

I still want to remove VS 2019, but I'd like to do it separately if we approve doing so since there is work beyond just deleting this project. We can also simplify the build process and must change the cli install command code to drop it.

Reply:
#1130